MapboxNavigation/RouteVoiceController.swift

```diff
@@ -216,6 +218,8 @@ open class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate, AVAudioP
      - parameter instruction: The instruction to read aloud.
      */
     open func speak(_ instruction: SpokenInstruction) {
+        guard let routeProgress = routeProgress else { return }
```

Comment:
Instead of failing silently, assert that there’s a route progress. Better to fail early knowing why than to fail silently without any clues leading to the root cause.